src/controllers/post/getExplorePosts.ts

```diff
@@ -0,0 +1,22 @@
+import { type Request, type Response, type NextFunction } from 'express'
+import { getExplorePostsQuery } from '../../queries/post'
+import CustomError from '../../helpers/CustomError'
+
+const getExplorePosts = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+  try {
+    const { page } = req.query
+    const pageNumber = Number(page)
+    if (pageNumber < 0 || Number.isNaN(pageNumber)) {
```

Comment:
Without seeing the code I would never guess that you need to send page number as a query string
so you either should response with a message that asks for the page number or give a default page number like this
```suggestion
    const { page } = req.query
    const pageNumber = Number(page) | 1
    if (pageNumber < 0 || Number.isNaN(pageNumber)) {
```